src/app/tabs/compile-tab.js

```diff
@@ -14,9 +14,9 @@ var remixLib = require('remix-lib')
 var styleGuide = remixLib.ui.themeChooser
 var styles = styleGuide.chooser()
 
-function compileTab (container, appAPI, appEvents, opts) {
-  if (typeof container === 'string') container = document.querySelector(container)
-  if (!container) throw new Error('no container given')
+function compileTab (api = {}, events = {}, opts = {}) {
+  var appAPI = api
+  var appEvents = events
```

Comment:
same here, well that's fine to keep `appEvents`, or change every occurence of `appEvents` to `events`